tests/test_response_handler.py

```diff
@@ -2,7 +2,12 @@
 
 from cornice_swagger.swagger import ResponseHandler, CorniceSwaggerException
 from cornice_swagger.converters import convert_schema
-from .support import BodySchema, HeaderSchema, ResponseSchema, response_schemas
+from .support import (BodySchema,
```

Comment:
I was trying to follow the cornice way to do it, but I just realized in [some parts](https://github.com/Cornices/cornice/blob/master/cornice/pyramidhook.py#L8) they have all kinds of styles. So it's OK to keep it like this. I just think it's important to make the rest of the package consistent.